README.md

```diff
@@ -13,7 +13,7 @@ This file should be placed in the root of the repository and set to your liking.
 
 ```
 {
-    "schedule": "daily", // daily|weekly|monthly
+    "Schedule": "Daily", // daily|weekly|monthly
```

Comment:
This is not going to parse correctly if used in a config file.
If you want this to accept uppercase "Daily" we will need to update the code in [Schedule.cs](https://github.com/dabutvin/Imgbot/blob/master/CompressImagesFunction/Schedule.cs)
And [add a test](https://github.com/dabutvin/Imgbot/blob/master/Test/ScheduleTests.cs)

What do you think?